partner_invoicing_mode/views/res_config_settings.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="inherit_id" ref="account.res_config_settings_view_form" />
+        <field name="model">res.config.settings</field>
+        <field name="arch" type="xml">
+            <xpath expr="//block[@id='invoicing_settings']" position="inside">
+                <setting
+                    id="invoicing_mode"
+                    string="Invoicing Mode"
+                    help="Standard Invoicing Options"
+                >
+                    <div class="mt16">
+                        <div class="content-group" id="standard_invoicing_settings">
+                            <div>
+                                <field name="invoicing_mode_standard_last_execution" />
+                                <label
```

Comment:
Please double-check position of `invoicing_mode_standard_last_execution `.
![Ảnh chụp Màn hình 2024-08-06 lúc 10 48 50](https://github.com/user-attachments/assets/ba7384d1-82f2-4922-acac-53b791006ca0)